src/tests/EShopworld.WorkerProcess.UnitTests/LeaseAllocatorTests.cs

```diff
@@ -61,8 +61,9 @@ public async Task TestAllocateLeaseAsyncExistingActiveLease()
             result.Should().BeNull();
         }
 
+
         [Fact, IsUnit]
-        public async Task TestAllocateLeaseAsyncExistingExpiredLeaseLeaseCreated()
+        public async Task TestAllocateLeaseAsyncExistingExpiredLeaseShouldUpdateLease()
```

Comment:
Can we rename the tests so they follow the naming convention:
- `[subject at test]_[condition]_[result]`

So this test would be:
AllocateLease_ExistingExpiredLeaseAndWinning_UpdateExistingLease

Reply:
Naming updated